Source/JNA/pom.xml

```diff
@@ -202,6 +202,8 @@
                     <configuration>
                         <!-- Slightly faster builds, see https://issues.apache.org/jira/browse/MCOMPILER-209 -->
                         <useIncrementalCompilation>false</useIncrementalCompilation>
+                        <source>1.8</source>
```

Comment:
cool thanks.  you have sparked a fire.  I'm going to go ahead and start on 1.9.  I aleady had a 1.8.x branch saved off so I'll just cherry pick for needs over there.  Appreciate it very much!